improver_tests/acceptance/test_apply_emos_coefficients.py

```diff
@@ -350,24 +331,6 @@ def test_wrong_land_sea_mask(tmp_path):
         run_cli(args)
 
 
-def test_wrong_forecast_coefficients(tmp_path):
```

Comment:
What happens if you do run this test?  Presumably some error is raised?  It would be good to keep coverage of the case where coefficients are provided but the input forecast file is not.

Reply:
I've added this test back in and just caught the error that arises. In this example, the error is coming from Iris, as we're providing a CubeList for the input argument when a Cube is expected.